docker/ghd

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Link me into your $PATH as `ln -s ../path/to/ghd`
+
+script_path() {
+    if [[ "$OSTYPE" == "darwin"* ]]; then
+        READLINK="readlink"
+    else
+        READLINK="readlink -f"
+    fi
+
+    echo "$(dirname "$($READLINK "${BASH_SOURCE[0]}")")"
+}
+
+current_dir="$(pwd -P)"
+
+pushd "$(script_path)" > /dev/null
+
+    set +e
+
+    poetry run bash -c " cd \"$current_dir\" ; ghd $@ "
+
+popd > /dev/null
```

Comment:
This shouldn't be necessary unless this file is `source`d, as the working directory is local to the shell.

Reply:
Ok, then I can remove `pushd` at the top and replace it with `cd`...